server/cmd/mmctl/commands/sampledata.go

```diff
@@ -179,8 +182,12 @@ func processProfileImagesDir(profileImagesPath, tmpDir, bulk string) ([]string,
 		// we're not importing the resulting file, so we keep the
 		// image paths corresponding to the value of the flag
 	} else {
-		for _, profileImage := range profileImagesFiles {
-			profileImages = append(profileImages, filepath.Join(profileImagesPath, profileImage.Name()))
+		for _, dirEntry := range dirEntries {
+			fileInfo, err := dirEntry.Info()
```

Comment:
Why is `fs.FileInfo` needed here? Is `os.DirEntry` enough?

Reply:
Previously ioutil.ReadDir was returning FileInfo[] but os.ReadDir returns DirEntry[] so I wanted to keep the existing return type without breaking anything.
https://pkg.go.dev/io/ioutil#ReadDir
https://pkg.go.dev/os#ReadDir

Reply:
Would you please check if `DirEntry[]` is enough here? The code will fail to compile if any field is missing.

Reply:
The build was successful when i ran locally.
In the docs also return type is mentioned as []DirEntry and pipelines are also successful👍

Reply:
Awesome to hear that `DirEntry` is enough here. Could you also check the other code pieces where you migrated `ioutil.ReadDir` and check if `DirEntry` is enough there?